ipn/ipnlocal/local.go

```diff
@@ -3128,8 +3128,8 @@ func peerAPIBase(nm *netmap.NetworkMap, peer *tailcfg.Node) string {
 	}
 	var p4, p6 uint16
 	svcs := peer.Hostinfo.Services()
-	for i, n := 0, svcs.Len(); i < n; i++ {
-		s := svcs.At(i)
+	for i := svcs.Iterator(); i.Valid(); i.Next() {
```

Comment:
How about just `Iter()`. And I guess `Valid` works, but something even shorter like `Ok` is tempting.

And it'd be nice of the whole thing were shorter somehow. Like renaming Valid to Next and making Next advance, so it operates like https://pkg.go.dev/database/sql#Rows.Next and dropping the final PostStmt.